### ISSUE TITLE
Add Romanian parcel locker Ship&Go by Cargus

### ISSUE THREAD

**Issue by DnaX** [osmlab/name-suggestion-index#7502]
The logistic operator Cargus started to install lots of parcel lockers branded Cargus Ship&Go: https://www.cargus.ro/en/point-location/ and https://www.cargus.ro/en/cargus_launches_its_parcel_lockers-_network/

Suggested tagging:

```
amenity=parcel_locker
brand=Ship&Go
brand:wikidata=t.b.d.
name=Cargus Ship&Go
operator=Cargus
operator:wikidata=Q12723442
```

**Comment by DnaX** [comment]
Also, the name should be preservered.

**Comment by DnaX** [comment]
@Dimitar5555 here now: https://www.wikidata.org/wiki/Q117327750

I'm in dubt for "Ship&Go" or "Ship & Go". Following the website naming seems better the latter: https://www.cargus.ro/en/ship-and-go-en/

**Comment by Dimitar5555** [comment]
Resolved by 585b58f, I've used `Ship & Go` and added `Ship&Go` as `matchNames`.
